packages/elastic-datemath/src/index.d.ts

```diff
@@ -36,7 +36,7 @@ declare const datemath: {
     input: string,
     options?: {
       roundUp?: boolean;
-      forceNow?: boolean;
+      forceNow?: Date;
```

Comment:
not familiar with this code, but could we rename parameter or add some jsdocs decribing what this forceNow does ?